sympy/sets/conditionset.py

```diff
@@ -18,30 +19,31 @@ class ConditionSet(Set):
 
     >>> from sympy import Symbol, S, ConditionSet, Lambda, pi, Eq, sin, Interval
     >>> x = Symbol('x')
-    >>> sin_sols = ConditionSet(Lambda(x, Eq(sin(x), 0)), Interval(0, 2*pi))
+    >>> sin_sols = ConditionSet(x, Eq(sin(x), 0), Interval(0, 2*pi))
     >>> 2*pi in sin_sols
     True
     >>> pi/2 in sin_sols
     False
     >>> 3*pi in sin_sols
     False
-    >>> 5 in ConditionSet(Lambda(x, x**2 > 4), S.Reals)
+    >>> 5 in ConditionSet(x, x**2 > 4, S.Reals)
     True
     """
-    def __new__(cls, condition, base_set):
-        if condition.args[1] is S.false:
+    def __new__(cls, vari, condition, base_set):
```

Comment:
> Why vari, why not simply var?

We have a global `var`(`sympy.core.symbol.var`) that's why. So i think it may create a confusion . Is not it ?